src/components/dar_collection_table/DarCollectionTable.js

```diff
@@ -103,28 +103,67 @@ const CancelCollectionButton = (props) => {
   });
 };
 
+const ResubmitCollectionButton = (props) => {
+  const { collection } = props;
+  return h(SimpleButton, {
+    keyProp: `resubmit-collection-${collection.id}`,
+    label: 'Revise',
+    //todo: should we filter out any statuses that could be resubmitted? is this the correct place for such logic?
+    //disabled: includes(determineCollectionStatus(collection))(['Under Election']),
```

Comment:
I don't think this is necessary here. I can't think of a case where we want to show a disabled "Revise" button. I think that if that were the case, this button should not be on the screen at all.